### ISSUE TITLE
Mouse-wheel without the alt key still changes zoom.

### ISSUE THREAD

**Issue by zaun** [lospec/pixel-editor#103]
**Describe the bug**
The zoom changes on scroll-wheel without the alt key pressed.

**To Reproduce**
1. Go to the editor
2. Use the mouse scroll-wheel

**Expected behavior**
Not to scroll

**Desktop (please complete the following information):**
 - OS: MacOS Monterey 12.5
 - Browser Chrome 
 - Version 106.0.5249.103

**Additional context**
As an alternative if there was a way to turn off mouse-wheel zoom all together. With the keyboard zoom commands, mouse-wheel isn't really needed.

**Comment by unsettledgames** [comment]
We're considering adding back the zoom tool, however I can't replicate the bug on chrome. Are you also pressing ctrl or do you have any particular key shortcuts enabled on your pc perhaps?

**Comment by zaun** [comment]
No, it could well be the mouse. I'm on a mac with the Magic Mouse and it's mouse wheel function is overly sensitive.

**Comment by unsettledgames** [comment]
Sorry, got confused with another bug. We'll probably add back the zoom tool, which can be used by clicking. We'd also like to disable zoom on the page and enable ctrl+ / ctrl-

**Comment by zaun** [comment]
Allowing ctrl+ / ctrl- to zoom the editor would be best, having to click-to-zoom works as well but not as nice as keyboard controls. But as it it, the editor is basically unusable for me with my current setup. It constant changes zoom while just trying to edit.

**Comment by unsettledgames** [comment]
Currently working on it in the bug-fixes banch
